hack/update-renovate-baseBranches.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I prefer `#!/usr/bin/env bash`, but I see that some of our existing scripts also use this shebang